### PR TITLE
[Backport stable/8.3] Fix flaky test shouldResumeExporting in ExportEndpointIT

### PR DESCRIPTION
**Pull request by rodrigo-lourenco-lopes**
## Description

Backport of: https://github.com/camunda/zeebe/pull/14632

This PR fixes the flaky test shouldResumeExporting in ExportEndpointIT.

Since the cluster being used has more than one partition, it might happen that the start process command happens before the remaining partitions have received the process. That is because when a process gets deployed, it will be first sent to partition 1 and then forwarded to the remaining partitions.

Since the test here is only trying to assert that exporting gets resumed, then it is ok to just publish a message that will generate a record instead of deploying a process.
## Related issues

closes #

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java [hunk 1]
```diff
@@ -98,7 +104,13 @@ void shouldResumeExporting() {
...
 
     deployProcess(client);
-    startProcess(client);
+    client
+        .newPublishMessageCommand()
+        .messageName("Test")
+        .correlationKey("1")
+        .messageId("2")
+        .send()
+        .join();
```
Comment (deepthidevaki) [inline qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java:113]: :wrench: Can extract this into a method to reuse in both tests.

**Review by deepthidevaki — APPROVED** [review]
:rocket:

**Comment by rodrigo-lourenco-lopes** [comment]
bors merge

**Comment by rodrigo-lourenco-lopes** [comment]
bors retry

**Pull request merged**